src/app/login/login.component.html

```diff
@@ -25,6 +25,10 @@
             </div>
             <div class="col-md-4">
                 <input type="submit" id="btn-login" class="btn btn-custom btn-lg btn-block" value="Log in" [disabled]="!loginForm.form.valid">
+                <br>
+                <a href="#signup" id="btn-login" class="btn btn-custom btn-lg btn-block btn-success">sign up</a>
```

Comment:
Any particular reason of not using `routerLink` here for redirection of routes?
If not, then I'd recommend using `routerLink` for page routing.